packages/cli/src/voting/commitVotes.js

```diff
@@ -110,9 +122,14 @@ const commitVotes = async (web3, oracle, designatedVoting) => {
         );
         console.group(style.success("Receipts:"));
         successes.forEach(committedVote => {
+          const identifierNonStandardPrecision = IDENTIFIER_NON_18_PRECISION[
```

Comment:
nit: I think this variable name is a little misleading -- shouldn't it just be `identifierPrecision` as it is elsewhere?

Reply:
Done